robotoff/app/api.py

```diff
@@ -1141,6 +1142,38 @@ def on_get(self, req: falcon.Request, resp: falcon.Response):
         resp.media = response
 
 
+class ImagePredictionCollection:
+    def on_get(self, req: falcon.Request, resp: falcon.Response):
+        response: JSONType = {}
+        count: int = req.get_param_as_int("count", min_value=1, default=25)
+        page: int = req.get_param_as_int("page", min_value=1, default=1)
+        with_logo: Optional[bool] = req.get_param_as_bool("with_logo", default=False)
+        barcode: Optional[str] = req.get_param("barcode")
+        type: Optional[str] = req.get_param("type")
```

Comment:
We want to be compatible with parameters from InsightCollection, so we want a list of type.

```suggestion
        keep_types: Optional[List[str]] = req.get_param_as_list(
            "types", required=False
        )
```

This means changing get_image_predictions code